protected/humhub/modules/user/controllers/AuthController.php

```diff
@@ -118,6 +121,7 @@ public function actionLogin()
             'model' => $login,
             'invite' => $invite,
             'canRegister' => $invite->allowSelfInvite(),
+            'passwordRecoveryRoute' => $this->module->passwordRecoveryRoute,
```

Comment:
There is also a second view file `return $this->renderAjax('login_modal', $loginParams);` which has a "Password recovery" link. Can you please add the condition also there.